src/components/screens/PointOfInterest.js

```diff
@@ -210,11 +210,20 @@ export const PointOfInterest = ({ data, hideMap, navigation, route }) => {
                 [category.iconName || MAP.DEFAULT_PIN]: 1,
                 iconName: category.iconName || MAP.DEFAULT_PIN,
                 id,
-                position: { latitude, longitude }
+                position: { latitude, longitude },
+                serviceName: title,
+                title
               }
             ]}
-            selectedMarker={id}
             mapStyle={styles.mapStyle}
+            interactivity={{
```

Comment:
I can add new pins to the POI map. This shouldn't happen. I added the red marked pin in the screenshot myself.

<img width="156" alt="Bildschirmfoto 2025-05-13 um 09 32 17" src="https://github.com/user-attachments/assets/e5924a0d-4025-45cd-aac5-dcd011c509eb" />

Reply:
I added `onMapPress` control to `handleMapPressToSetNewPin` function and the problem is solved.